pkgs/development/web/nodejs/v23.nix

```
@@ -53,5 +53,15 @@ buildNodejs {
       hash = "sha256-gmIyiSyNzC3pClL1SM2YicckWM+/2tsbV1xv2S3d5G0=";
       revert = true;
     })
+    # Fix for https://github.com/NixOS/nixpkgs/issues/355919
+    # FIXME: remove after a minor point release
+    (fetchpatch2 {
+      url = "https://github.com/nodejs/node/commit/a094a8166cd772f89e92b5deef168e5e599fa815.patch?full_index=1";
+      hash = "sha256-5FZfozYWRa1ZI/f+e+xpdn974Jg2DbiHbua13XUQP5E=";
+    })
+    (fetchpatch2 {
+      url = "https://github.com/nodejs/node/commit/7f40deae6fee1534512ce25b25f9b89ec7e96aee.patch?full_index=1";
```

```suggestion
      url = "https://github.com/nodejs/node/commit/f270462c09ddfd770291a7c8a2cd204b2c63d730.patch?full_index=1";
```
https://github.com/nodejs/node/pull/55908 has landed in https://github.com/nodejs/node/commit/f270462c09ddfd770291a7c8a2cd204b2c63d730 which should be preferred over commit from PR.

---

Thanks